paper/main_sections/regression.tex

```diff
@@ -1,65 +1,104 @@
-is to preform linear regression. Initially, we just used the convoluted predicted
-hemodyamic response (HR) using all the conditions together and each of 
-conditions individually. After realizing that the HRs themselves didn't explain
-enough of the BOLD ratio, attended to add features in order to reduce/ explain
-the noise we observed. Additional features that we examined included a 
-linear drift, a small set the fourier series, and a few principle components.
-
-Linear regression suggests a linear relationship between an observation's response,
-$y_i$ with it's features $x_i$ (a vector who's first element is $1$ and the rest are
-the values the observation $i$'s features are) in the following manner:
-
-\begin{equation}
-y_i = x_i^T \beta + \epsilon_i
-\end{equation}
-
-where $\epsilon_i \sim N(0, \sigma^2)$. Moveover we assume for each observation
-($i$), the $\epsilon_i$ are independent and that the true $\beta$ value is the same.
-As such we can rewrite the relationship we assume is correct in a matrix form:
+\par \indent A simple and straightforward way to model the voxel time courses 
+for each subject is to perform linear regression. Initially, we just used the 
+convolved predicted hemodyamic response (HR) and either all the of the 
+conditions together or each of conditions individually. After realizing that 
+the HRs themselves didn't explain enough of the BOLD ratio, we attempted to 
+add features in order to reduce or explain the noise we observed. Additional 
+features that we examined included a linear drift and some of the time 
+courses' Fourier series and principal components.
+
+\par Linear regression assumes a linear relationship between a response 
+variable $Y$ and a design matrix of predictors $X$. Each element of $Y$ 
+represents a single observed response, and each row of $X$ represents a 
+corresponding vector of predictor values. If including the intercept as a 
+term (as we have elected to do), the first column of the $X$ design matrix 
+should be a vector of $1$s. The linear model can then be expressed as: 
 
 \begin{equation}
 Y = X\beta + \epsilon
 \end{equation}
 
-This relationship requires a $X$ feature matrix, which presents each distinct feature
-as a column and sees each observation get a single row to represent it's values.
-Assuming $\epsilon \sim N(0,\sigma^2 I)$ isn't automatic, so we will need to check 
-the validity of this assumptions, as discussed in the \textit{"Normality Assumptions"} 
-section. As might be expected, we do not know the true value of $\beta$ and as such
-need to estimate, using matrix algebra we get a closed from solution:
+\par It is further assumed that the errors $\epsilon_i$ for each observation 
+$i$ are independent and identically distributed with $N(0, \sigma^2)$, and 
+that the errors are independent of $X$. The coefficients $\beta$ can be 
+estimated with the closed-form solution:
 
 \begin{equation}
 \hat{\beta} =(X^T X)^{-1} X^T 
```

Comment:
I swear I corrected this. Can you make sure it says (X^T X)^-1 X^T y      * include the y.